### PR TITLE
Use as package

### PR DESCRIPTION
**Pull request by stickeerehan**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
-    "email": "paul.tibbetts@stickee.co.uk"
-  }],
+  "authors": [
+    {
+      "name": "Paul Tibbetts",
+      "email": "paul.tibbetts@stickee.co.uk"
+    },
+    {
+      "name": "Rehan Aslam",
+      "email": "rehan.aslam@stickee.co.uk"
```
Comment (stickeepaul) [inline composer.json]: :wave:

File: README.md [hunk 2]
```diff
@@ -12,42 +10,31 @@ First install this package:
...
-parameters:
-  ...
-```
+### Husky (Task Runner)
 
-### PHPStan
-
-PHPStan can be ran using:
+It's advisable to use [Husky](https://github.com/typicode/husky) as the task runner. The config files 
+should already be published using the above command, however you will still need to install husky:
```
Comment (stickeepaul) [inline README.md]: change the above command to `yarn add stickee-javascript-code-style` and make that package run `npx install-peerdeps --dev eslint-config-stickee` on postinstall, and declare husky a dependency?

File: composer.json (hunk 1 above)
Comment (stickeerehan) [reply composer.json]: Ello

File: README.md (hunk 2 above)
Comment (stickeerehan) [reply README.md]: since husky is a task manager, i think it doesnt belong to the eslint package

File: composer.json [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
-    "nette/neon": "^3.0",
-    "symfony/yaml": "^4.2",
-    "symfony/console": "^4.2"
+    "friendsofphp/php-cs-fixer": "^2.15",
+    "sensiolabs/security-checker": "^6.0"
   },
   "autoload": {
     "psr-4": {
-      "Console\\": "src"
+      "Stickee\\": "src"
```
Comment (stickeegreg) [inline composer.json]: Can you change the root namespace from `Stickee` to `Stickee\LaravelCodeStyle` or something so it won't conflict with other modules

**Pull request merged**